README.md

```diff
@@ -3,39 +3,49 @@ Stockalike parts pack for Kerbal Space Program
 
-Now, 400 parts sounds like it would kill your PC. While the mod certainly is large, the memory overhead is likely lower than you would expect. Care has been taken to ensure the mod is efficient, with methods such as enforcing a strict texel density (a technique I have only used since mid-2017, so the older assets might not be as efficient) and texture atlasing to cram several parts onto each texture sheet.
+Now, 500 parts sounds like it would kill your PC. While the mod certainly is large, the memory overhead is likely lower than you would expect. Care has been taken to ensure the mod is efficient, with methods such as enforcing a strict texel density (a technique I have only used since mid-2017, so the older assets might not be as efficient) and texture atlasing to cram several parts onto each texture sheet.
 
-For those familiar with KSP mods, BDB can be considered a stockalike equivalent of FASA, and a US counterpart to Tantares' soviet rockets. 
+For those familiar with KSP mods, BDB can be considered a ReStockalike equivalent of FASA, and a US counterpart to Tantares' soviet rockets. 
 
-Currently, the mod includes (but is not limited to) parts to make:
+Currently, the mod includes (but is not limited to):
 - Mercury spacecraft (Courtesy of Bloeting Aerospace Corp)
 - Gemini spacecraft (Courtesy of Trails Manufacturing Co.)
+- Gemini Ferry
+- Big Gemini 69' (nice)
 - Apollo Block I and II spacecraft, as featured in REAL LIFE!
 - Apollo Block III, III+, IV, and V spacecraft, from Eyes Turned Skywards
 - AARDV "Aardvark" autonomous resupply vehicle, also from Eyes
-- Modular 1.875m station parts for Gemini-era space stations
+- Modular 1.875m and 1.5m station parts for Gemini-era space stations
 - Skylab station parts
 - Spacelab (from Eyes) station parts
+- Parts for the Apollo-Venus mission
 - Vanguard rocket and probe
 - Scout rocket
 - Redstone rocket, with Sergeant clusters and Explorer 1
 - Juno II, IV rockets
-- Diamant rocket (All variants)
 - Thor-Delta rocket (Almost all variants can be made)
-- Delta 2 rocket (Including upper stage and a variety GEM SRBs)
+- Delta 2 and 3 rockets (Including upper stage and a variety of GEM SRBs)
+- 4 meter Delta Cryogenic Second Stage, modeled by Blowfish. (5 meter available in ReDIRECT)
 - Atlas rocket (everything from Atlas A up through Atlas V 551)
 - Agena upper stage (All variants)
 - Vega (old Kerolox upper stage for Atlas)
 - Centaur D, G, G', T, V upper stages
 - Titan 1, 2, 3, 4 rockets, with SRMs and Transtage (No IUS yet!)
```

Comment:
We have both IUS and the smaller TOS now.

Reply:
Already mentioned lower down; but i do need to remove that, thanks!